### ISSUE TITLE
Table search+filter clean up

### ISSUE THREAD

**Issue by ericsoderberghp** [grommet/hpe-design-system#1583]
Aggregator for a pile of issues.

- [x] filter + selection summary  - site (waiting on Matt)
- [ ] table example [1282](https://github.com/grommet/hpe-design-system/issues/1282) [1343](https://github.com/grommet/hpe-design-system/issues/1343) - site 
- [x] refactor examples [1122](https://github.com/grommet/hpe-design-system/issues/1122) - site (waiting on Matt)
- [x] search summary [1348](https://github.com/grommet/hpe-design-system/issues/1348) - site  (waiting on Matt)
- [x] filter badge count [1347](https://github.com/grommet/hpe-design-system/issues/1347 
- [x] Grommet issue pr to be consumed by the DS. add < Button badge={} />
- [x] filter chips [1414](https://github.com/grommet/hpe-design-system/issues/1414) -> Keep within Figma in the Leftovers tab. Remove from the DS site (Brittany)
- [x] filter button outline [1118](https://github.com/grommet/hpe-design-system/issues/1118) -> add < Button kind="X" /> < Menu kind="X" /> (THIS WILL WORK THE SAME AS `PASSING KIND="X"` AS IT DOES WITH `BUTTON`)
- [x] filtering with range selector [1346](https://github.com/grommet/hpe-design-system/issues/1346) On hold. Looking for clarification from Vicky.

**Comment by vavalos5** [comment]
Ticket #1346 is no longer needed. I closed that ticket :)

**Comment by halocline** [comment]
Closing via https://github.com/grommet/hpe-design-system/pull/1619 plus the following:

Creating the following issues to close out the remainder of this ticket:
- https://github.com/grommet/hpe-design-system/issues/1282
- https://github.com/grommet/hpe-design-system/issues/1631
- https://github.com/grommet/hpe-design-system/issues/1630

**Comment by halocline** [comment]
See previous comment
